app/workers/reactions/bust_reactable_cache_worker.rb

```diff
@@ -0,0 +1,21 @@
+module Reactions
+  class BustReactableCacheWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :high_priority, retry: 10
+
+    def perform(reaction_id)
+      cache_buster = CacheBuster
```

Comment:
I know this was done since we removed the injected dependency from the arguments, but assigning the class to a variable instead of just calling the class method ends up being slightly confusing. Let's maybe remove this indirection, we're not using it elsewhere in the code base.

Reply:
Sure @citizen428! Thanks for reviewing.

Reply:
Sure @citizen428! Thanks for reviewing